pages/regions.js

```diff
@@ -0,0 +1,65 @@
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import { nanoid } from 'nanoid';
+
+import Layout from '../src/components/layout';
+import { Hero, Icon, P } from '../src/components/jambonz-ui';
+import { getData } from '../src/lib/data';
+
+function Regions({ data }) {
+  const router = useRouter();
+  const redirect = router.query.redirect;
+
+  return (
+    <>
+      {data.regions.map((region) => {
+        const href = (redirect === 'login') ? region.url : region.altUrl;
+
+        return (
+          <div key={nanoid()} region={region.name} className="wrap regions__wrap">
+            <div className="regions__icon">
+              <Link href={href}>
+                <a>
+                  <Icon name={region.icon} subStyle={region.color} mainStyle="fill" />
+                </a>
+              </Link>
+            </div>
+            <P className='h6'>
+              <span style={`color:${region.color}`}>{region.title}</span>
+            </P>
+            <div className="i">
+              <Link href={href}>
+                <a>
+                  <P className='h6'>
+                    <span style='color:black'>{region.text}</span>
```

Comment:
No inline styles please. You can create a selector in your regions SASS file that can target this and apply the color using our SASS variables. We should always style in the SASS using our variables so the colors are consistent.

The color you're looking for here is:

https://github.com/jambonz/next-static-site/blob/main/src/styles/_vars.scss#L14

Yes, it is just the hex for black. But, if we ever wanted to change that to a slightly off black tone we can ensure all typography across the web app will adjust properly. Rendering inline styles like this makes things unpredictable.